config/locales/en.yml

```diff
@@ -104,8 +104,9 @@ en:
     inline_feedback_error: Please select an option.
```

Comment:
I appreciate that this is a large file with some potential sortability issues.  So, instead of trying to sort it as part of this PR, could you please put a ticket in the backlog to cover refactoring it?  Something like "increase the ease of maintenance of the `en.yml` file", or some such (Breaking it out into separate files would be a reasonable approach).

Reply:
Done